Updated get started and generic procedures for types and streams (#464)
osisoft/OCS-Docs: content/overview/get-started/gs-streams.md

## Procedure

1. Click the ![Menu icon](images/menu-icon.png) and click **Sequential Data Store** (under Data Management).

1. Click the **Namespace** selection button to open the `Select a Namespace` window. Select the namespace in which you want to create the stream. 
    ![Namespace list](images/sds-gs-namespace.png)

1. From the **Streams** drop-down list, select **Streams** if it is not already selected. 
    ![Streams list](images/streams-list.png)

1. Click the ![More options](images/more-options.png) icon on the menu bar and click **Manage Default Permissions**.  
    The `Manage Default Permissions for New Streams` window opens. The default permissions for streams created in the selected namespace are configured here.
   
    **Note:** By default, you must be assigned the Tenant Administrator role to change the default permissions. Any changes only affect new streams. It does not change the permissions on already existing streams. 
    

>     When you are done reviewing the permissions, click **Cancel** to continue.

Author: ```suggestion
    When you are done reviewing permissions, click **Cancel** to continue.
```

> 1. Click the ![Menu icon](images\menu-icon.png) and click **Sequential Data Store** (under Data Management).

Author: ```suggestion
1. Click the ![Menu icon](images/menu-icon.png) and click **Sequential Data Store** (under Data Management).
```

1. Click the **Namespace** selection button to open the `Select a Namespace` window. Select the namespace in which you want to create the stream. 
    ![Namespace list](images/sds-gs-namespace.png)

1. From the **Streams** drop-down list, select **Streams** if it is not already selected. 
    ![Streams list](images/streams-list.png)

1. Click the ![More options](images/more-options.png) icon on the menu bar and click **Manage Default Permissions**.  

>     This opens the `Manage Default Permissions for New Streams` window where default permissions for streams created in the selected namespace are configured. 

Author: ```suggestion
    The `Manage Default Permissions for New Streams` window opens. The default permissions for streams created in the selected namespace are configured here.
```

>     **Note:** By default, you must be assigned the Tenant Administrator role to change the default permissions. Any changes only affect new streams. It does not change the permissions on already existing steams. 

Author: ```suggestion
    **Note:** By default, you must be assigned the Tenant Administrator role to change the default permissions. Any changes only affect new streams. It does not change the permissions on already existing streams. 
```

1. In the `Add Stream` window, enter the following values for these fields:

   - `Id` &ndash; **MyOCS.NorthAmerica.SLTC.PumpA**
   - `Description` &ndash; **SDS Stream used by My OCS**
   - `Type` &ndash; **MyOCS.PumpState**

    **Note:** Leave the `Name` field blank. By default, it will take the value of the Stream Id.

1. Click the **Tags** tab, if it is not already selected.

1. In the `New Tag` field, type **OSIsoft**, and click **+** to add the tag. 

    **Note:** You can click the *X* to delete it from the stream.

1. Click the **Metadata** tab and click **Add Metadata**.

1. In the `Metadata Key` field,  enter **Site**, and in `Metadata Value`, enter **SLTC**. 

    **Note:** You can click the *X* to delete them from the stream. Once the stream is created, you can select the stream and click **Edit** to make changes to the tags and the metadata.

1. Click **Save**.

1. In the `Search` field, enter **Site:SLTC**

>     This searches for streams that have the metadata key *Site* and the value *SLTC*. This search query returns the MyOCS stream. 

Author: ```suggestion
    OCS searches for streams that have the metadata key *Site* and the value *SLTC*. This search query returns the MyOCS stream. 
```
